src/auth/adapters/facebook.clj

```diff
@@ -9,3 +9,11 @@
    :register/cred-type :credential.type/facebook
    :register/email     (:fb-user/email fb-user)
    :register/fb-id     (:fb-user/id fb-user)})
+
+
+(s/defn external-fb-user->internal-fb-user :- wire.fb-user/FbUser
+  [external-fb-user]
+  {:fb-user/id          (:id external-fb-user)
+   :fb-user/email       (or (:email external-fb-user) "unknwon@unknown.com")  ; FIX THIS
```

Comment:
fix this 😂

Reply:
The problem is that some users on facebook do not have an e-mail registered (that is my case)
I don't know how we can deal with this... It would be a complicated work between the frontend and backend (because the user would have to enter the e-mail manually).